repo/packages/B/beta-tensorflow/1/config.json

```diff
@@ -0,0 +1,329 @@
+          "default": ""
+        },
+        "virtual_network_enabled": {
+          "description": "Enable virtual networking",
+          "type": "boolean",
+          "default": false
+        },
+        "virtual_network_name": {
+          "description": "The name of the virtual network to join",
+          "type": "string",
+          "default": "dcos"
+        },
+        "virtual_network_plugin_labels": {
+          "description": "Labels to pass to the virtual network plugin. Comma-separated key:value pairs. For example: k_0:v_0,k_1:v_1,...,k_n:v_n",
+          "type": "string",
+          "default": ""
+        },
+        "log_level": {
+          "type": "string",
+          "description": "The log level for the DC/OS service.",
+          "default": "INFO",
+          "enum": [
+            "OFF",
+            "FATAL",
+            "ERROR",
+            "WARN",
+            "INFO",
+            "DEBUG",
+            "TRACE",
+            "ALL"
+          ]
+        },
+        "job_url": {
+          "title": "Job URL",
+          "description": "URL for the TensorFlow job you want to run on DC/OS (ZIPs are okay).",
+          "type": "string",
+          "default": ""
+        },
+        "job_path": {
+          "title": "Job Path (Optional)",
+          "description": "Path to your TensorFlow job (assume job_url has already been downloaded/unpacked to the current directory). This path will be added to PYTHONPATH at runtime.",
+          "type": "string",
+          "default": ""
+        },
+        "job_name": {
+          "title": "Job Name",
+          "description": "Name of the module (without extension) whose main function should be executed.",
+          "type": "string",
+          "default": ""
+        },
+        "job_context": {
+          "title": "Job Context (Optional)",
+          "description": "A dictionary to pass to your job's main function (intended for run-time configuration options like batch_size, learning_rate, etc.).",
+          "type": "string",
+          "default": "{}"
+        },
+        "shared_filesystem": {
+          "title": "Shared Filesystem",
+          "description": "Distributed TensorFlow relies on a shared filesystem (GCS, HDFS, etc.) for summaries/checkpoints. Please set up one of these filesystems before using this package, and specify the location here (eg. 'gs://your/directory' or 'hdfs://namenode:port/your/directory').",
+          "type": "string",
+          "default": ""
+        },
+        "gcs_key": {
+          "title": "GCS Key",
+          "description": "Stringified contents of JSON key for Google Cloud Storage service account. Only use this field if use_gcs_key_secret is set to False.",
+          "type": "string",
+          "default": ""
+        },
+        "use_gcs_key_secret": {
+          "description": "A DC/OS secret called gcs_key exists and should be consumed by this framework.",
```

Comment:
It feels a bit weird to hard-code the keyname as gcs_key by default. I would expect this to be a string field with the *name* of the key to use.